config/aaa.py

```diff
@@ -256,7 +285,8 @@ def passkey(ctx, secret):
 @click.option('-o', '--port', help='TCP port range is 1 to 65535, default 49', type=click.IntRange(1, 65535), default=49)
 @click.option('-p', '--pri', help="Priority, default 1", type=click.IntRange(1, 64), default=1)
 @click.option('-m', '--use-mgmt-vrf', help="Management vrf, default is no vrf", is_flag=True)
-def add(address, timeout, key, auth_type, port, pri, use_mgmt_vrf):
+@click.option('-e', '--encrypt', help='Enable passkey encryption feature', is_flag=True)
```

Comment:
>encrypt [](http://example.com/codeflow?start=23&length=7)

This command line extra option is confusing, why you want to change TACPLUS/global/key_encrypt in 2 command lines? do you want to encrypt other passkey if the 2 commands are both used but with different encrypt option?